src/renderer/gdi/math.cpp

```diff
@@ -18,12 +18,7 @@ using namespace Microsoft::Console::Render;
 // - S_OK or math failure
 [[nodiscard]] HRESULT GdiEngine::GetDirtyArea(gsl::span<const til::rect>& area) noexcept
 {
-    auto rc = _psInvalidData.rcPaint;
-
-    SMALL_RECT sr = { 0 };
-    RETURN_IF_FAILED(_ScaleByFont(&rc, &sr));
-
-    _invalidCharacters = til::rect{ sr };
+    _invalidCharacters = til::rect{ _psInvalidData.rcPaint }.scale_down(_GetFontSize());

```

Comment:
i hate so much how we had two functions with the same name that did OPPOSITE THINGS. thanks for fixing